client/src/views/student/MySubjectDetail.vue

```diff
@@ -82,21 +92,42 @@ export default {
             this.fullscreenLoading = false;
             this.$swal('Enrolled!', 'Successfully enrolled this class.', 'success');
           }
+          await this.getSubject(this.$route.params.subjectId);
+        }
+      });
+    },
+    async cancelClass(row) {
+      this.$swal({
+        text: 'Are you sure to cancel enrollment this class ?',
+        type: 'success',
+        showCancelButton: true,
+        cancelButtonColor: '#d33',
+        confirmButtonColor: '#28a745',
+        confirmButtonText: 'Confirm',
+        reverseButtons: true
+      }).then(async (result) => {
```

Comment:
đã khai báo hàm là `async` bên trên rồi thì dùng `await` để tạo ra cái `result` chứ lại dùng `then` làm gì nữa
`let result = await this.$swal....`